CHANGELOG.md

```diff
@@ -3,11 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unpublished Version / DEV]
+## [[3.2](https://github.com/nf-core/rnaseq/releases/tag/3.2)] - 2021-06-17
```

Comment:
You plan a release for tomorrow :drum:

Reply:
Oh, yes!! Not much left to do [now](https://github.com/nf-core/rnaseq/milestone/5)